recipe/meta.yaml

```diff
@@ -30,14 +30,14 @@ requirements:
     - python
     - pip
     - openmm
-    - plumed >=2.7
```

Comment:
Is this valid syntax?  If you're trying to pin to a particular version, shouldn't it be `==2.7`?

Reply:
Yes, it is correct. The tests run with the expected version.